draft-private-access-tokens.md

```diff
@@ -44,66 +44,98 @@ service to limit access according to the policy without tracking client identity
-enforcing policies. Typically, servers use these signals as weak client identifiers.
-Clients coming from a single IP address may be limited in how much content
-they can access over a given time period (often referred to as a "metered paywall"),
-or access from an IP address may be rate-limited to prevent fraud and abuse.
-When the IP address signal is unavailable, perhaps due to the use of a proxy network,
-servers are left without a suitable functional replacement.
-
-This document proposes using Private Access Tokens, using RSA Blind Signatures as defined
-in {{!BLINDSIG=I-D.irtf-cfrg-rsa-blind-signatures}} as a replacement for these signals. These
-tokens are privately issued to clients and then redeemed by servers in such a way that the
-issuance and redemption events for a given token are unlinkable. Fundamentally, using
-tokens in lieu of passive signals for policy enforcement seemingly requires some entity
-to know both the client and policy. However, with appropriate mediation and
-separation between parties involved in the issuance and redemption protocols,
-it is possible to limit this information leakage without functional regressions.
-
-This document describes a protocol for mediating the issuance and redemption
-of Private Access Tokens with the following properties:
-
-1. The Mediator enforces and maintains a mapping between Client identifiers
-   and anonymous Origin identifiers;
-1. The Issuer enforces policies keyed by anonymous Client identifier and Origin
-   identifier, without learning the real Client identity; and
-1. The Origin learns whether a given Client has a valid Private Access Token for
-   its policy.
+Servers commonly use passive and persistent identifiers associated with clients,
+such as IP addresses or device identifiers, for enforcing client and usage
+policies. For example, a server might limit access to the amount of content from
+an IP address over a given time period (referred to as a "metered paywall"), or
+a server might rate-limit access from an IP address to prevent fraud and
+abuse. Servers also commonly use the client's IP address as a strong indicator
+of the client's geographic location to limit access to services or content to a
+specific geographic area (referred to as "geofencing").
+
+However, passive and persistent client identifiers can be used by any entity
+that has access to it without the client's express consent. A server can use a
+client's IP address or its device identifier to track client activity. A
+client's IP address, and therefore its location, is visible to all entities on
+the path between the client and the server. These entities can trivially track a
+client, its location, and servers that the client visits.
+
+A client that wishes to keep its IP address private can hide its IP address
+using a proxy service. But doing so severely limits the client's ability to
+access services and content, since without a stable identifier, servers might
```

Comment:
```suggestion
access services and content, since without a stable and unique identifier, servers might
```

Reply:
Using a proxy service doesn't necessarily mean that the identifier is unstable. It could be stable, just shared amongst many others. (Hence my suggestion to add "and unique")